bumble/device.py

```diff
@@ -906,6 +906,30 @@ async def get_phy(self):
     async def request_remote_name(self):
         return await self.device.request_remote_name(self)
 
+    async def read_remote_feature(self) -> None:
```

Comment:
maybe this Connection method should be called `get_xxx` and delegate to a Device method, like `connection.get_phy` delegating to `device.get_connection_phy`, to follow the same pattern of locating the actual sending of HCI commands in the Device class.
In that case, the method could be `Connection.get_remote_le_features`, delegating to `Device.get_connection_remote_le_features`.

Reply:
I am think whether it's necessary to delegate methods(but I will do if you insist), since for most of the time, callers won't call via device. The benefit of this way I can come up is to make `Connection` class simpler.

Reply:
The idea is to keep the Connection class very simple, without direct knowledge of HCI commands, mostly as a helper class that's useful for grouping connection-related functions together, and keep the bulk of the actual implementation in the Device class. This pattern also allows some of the device class methods to use other properties of the Device object, without requiring the Connection class to know about the internal of Device (even though we don't really enforce any sort of public/private distinction between methods currently, maybe we will eventually as the implementation becomes more stable).

Reply:
Okay, moved implementation to Device.